### ISSUE TITLE
Feature Request: Grouping Chats

### ISSUE THREAD

**Issue by avronr** [vector-im/riotX-android#1670]
I know you can favorite chats but it would be better if we can group chats into various folders like work, projectx, personal etc. It would really help people to not to miss messages among the clutter.

**Comment by swapnilraj** [comment]
Communities seem to be the closest thing to this feature currently available in the matrix eco-system.

**Comment by avronr** [comment]
I'm aware of the communities feature and that's not what I was aiming for. It doesn't have to be shareable like Communities it could be just a feature on riot linked to the account.

**Comment by DonPrus** [comment]
Yes, there are times when I have a lot of people/chats in a community and I would like to group them somehow. I currently do this with favorites and low priority, but I'd like to collect the labels myself.

Like this in Slack:
![image](https://github.com/element-hq/element-android/assets/5755838/c85817d7-ad04-4805-98a5-2069c77c6816)
